portal/static/less/gil.less

```diff
@@ -2662,7 +2662,7 @@ input[type="search"] {
 .checkbox-inline input[type="checkbox"] {
     position: absolute;
     margin-left: -20px;
-    margin-top: 4px;
+    margin-top: 4px \9;
```

Comment:
This looks odd - should that `\9` really be there?

Reply:
Paul, that is css shorthand for IE9 and lower style